kaldb/src/test/java/com/slack/kaldb/chunk/ReadOnlyChunkImplTest.java

```diff
@@ -145,7 +147,8 @@ public void shouldHandleChunkLivecycle() throws Exception {
                 500,
                 0));
     assertThat(logMessageSearchResult.hits.size()).isEqualTo(10);
-    assertThat(readOnlyChunk.successfulChunkAssignments.count()).isEqualTo(1);
+    assertThat(meterRegistry.get(CHUNK_ASSIGNMENT_TIMER).tag("successful", "true").timer().count())
```

Comment:
Please add an assert for the failure timer count and eviction timer count to make sure that both those timers are not reported at the same time.

Reply:
Done.